bin/dependency_check.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Based on: https://github.com/jeremylong/DependencyCheck#docker
+
+set -e
+
+DC_VERSION="latest"
+DC_DIRECTORY=/tmp/OWASP-Dependency-Check
+CACHE_DIRECTORY="$DC_DIRECTORY/data/cache"
+DATA_DIRECTORY="$DC_DIRECTORY/data"
+GIT_ROOT=$(git rev-parse --show-toplevel)
+REPORT_DIRECTORY="$GIT_ROOT/odc-reports"
+VENV_DIRECTORY=$(mktemp -d)
```

Comment:
Add cleanup command. We have quite a lot of dependencies and temp venvs could fill up /tmp.

```
trap 'rm -rf "$VENV_DIRECTORY"' EXIT
```